src/routes.js

```diff
@@ -1,2 +1,4 @@
-export const ID_ROUTE = '/:type'
-export const HASH_ID_ROUTE = '/:type/:hashId'
+const ROUTE_PREFIX = process.env.PREACT_APP_HOST_PREFIX
+
+export const ID_ROUTE = `${ROUTE_PREFIX}/:type`
```

Comment:
these routes won't have issues with two slashes in the beginning?

Reply:
unfortunately, we do need the `/` at root so that it knows to use absolute paths. The // isn't great, but since it's just for dev purposes (and it works), it's not really a huge issue